woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -1959,6 +1963,7 @@ def test_set_types(sample_df):
     )
     assert sample_df.ww.index is None
     assert sample_df.ww.time_index is None
+    assert "custom_tag" in sample_df.ww.semantic_tags["is_registered"]
```

Comment:
This assertion should probably go above line 1958, since we're specifically checking that changing the logical type for a column doesnt remove the custom semantic tag.